Firestore/Example/Podfile

```diff
@@ -7,14 +7,14 @@ use_frameworks!
 platform :ios, '8.0'
 
 target 'Firestore_Example' do
-  pod 'FirebaseAuth', :path => '../../'
   pod 'FirebaseCore', :path => '../../'
   pod 'FirebaseFirestore', :path => '../../'
 
   target 'Firestore_Tests' do
     inherit! :search_paths
 
     pod 'FirebaseCore', :path => '../../'
+    pod 'FirebaseAuth', :path => '../../'
```

Comment:
I would expect that if this is required at all it would be required for the IntegrationTests.

Meanwhile I'm not sure I see the harm in just leaving the link to FirebaseBath within the example.

Reply:
I don't think FirebaseBath is publicly announced yet! :-P

Reply:
Alright, I'll revert this file.

> I don't think FirebaseBath is publicly announced yet! :-P

Looks like we've just *leaked* that. And given that the *plumbing* between it and the rest of firebase isn't there yet, we could end up getting all *wet*; it really needs to *soak* a bit more first. Well, at least the code is pretty *clean*.